package.json

```diff
@@ -64,6 +64,7 @@
     "events": "1.0.2",
     "exports-loader": "0.6.2",
     "express": "4.13.3",
+    "fg-loadcss": "1.3.1",
```

Comment:
You need to run `npm run update-deps` after adding a dependency. The `shrinkwrap` should have been updated automatically, but it seems it's acting up again.